project/ws/app/src/lib/routes/user-profile/components/user-profile/user-profile.component.html

```diff
@@ -416,7 +416,7 @@ <h3>{{'userProfile.updatePrimaryEmail' | translate }}</h3>
                           {{'userProfile.group' | translate }}
                         </label>
                         <mat-form-field appearance="outline" class="margin-top-s w-full" id="group">
-                          <mat-select placeholder="{{'userProfile.groupPlaceholder' | translate }}" id="group" #group i18n-placeholder="group"
+                          <mat-select placeholder="{{'userProfile.groupPlavceholder' | translate }}" id="group" #group i18n-placeholder="group"
```

Comment:
pls verify this variable name

Reply:
the same key name we are using in the multilingual. Next time we will fix it.